src/components/Exchange/ConfirmationBox.js

```diff
@@ -138,6 +140,9 @@ export default function ConfirmationBox(props) {
 
   const [allowedSlippage, setAllowedSlippage] = useState(savedSlippageAmount);
 
+  const existingPositionPriceDecimal = getPriceDecimals(chainId, existingPosition?.indexToken.symbol);
```

Comment:
existing position is optinal, but getPriceDecimals will throw an error if there is no token

Reply:
also i think existingPosition.indexToken === marketInfo.indexToken here

so we don't need to use existingPosition